DB/SharedConstants.lua

```diff
@@ -7,6 +7,12 @@ addonTable.constants = addonTable.constants or {}
 
 local C = addonTable.constants
 
+-- Types of items
+C.TOOLTIP = {
+	ACTIVATION_HOVER = "TIP_ACTIVATION_HOVER",
+	ACTIVATION_CLICK = "TIP_ACTIVATION_CLICK",
+}
```

Comment:
I think a slightly more accurate name would be helpful, like ``ACTIVATE_ON_HOVER`` or ``ACTIVATION_METHOD_HOVER``, or similar